core/src/main/java/org/springframework/security/authentication/DelegatingAuthenticationManager.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2002-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.authentication;
+
+import java.util.List;
+
+import io.micrometer.common.KeyValues;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.core.CredentialsContainer;
+import org.springframework.util.Assert;
+
+public final class DelegatingAuthenticationManager implements AuthenticationManager {
+
+	private final List<AuthenticationManager> delegates;
+
+	private ObservationRegistry registry;
+
+	private AuthenticationEventPublisher eventPublisher = new NullEventPublisher();
+
+	private KeyValuesProvider keyValuesProvider = new KeyValuesProvider();
+
+	public DelegatingAuthenticationManager(List<AuthenticationManager> delegates) {
+		Assert.notEmpty(delegates, "delegates cannot be null or empty");
+		this.delegates = delegates;
+	}
+
+	@Override
+	public Authentication authenticate(Authentication authentication) throws AuthenticationException {
+		ObservationContext context = new ObservationContext(authentication);
+		Observation observation = Observation.createNotStarted("spring.security.authentication", context, this.registry)
+				.keyValuesProvider(this.keyValuesProvider);
```

Comment:
The Observation needs to be started.
```suggestion
		Observation observation = Observation.createNotStarted("spring.security.authentication", context, this.registry)
				.keyValuesProvider(this.keyValuesProvider)
				.start();
```